client/js/components/fragments/Map.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+import GoogleMapsLoader from 'google-maps';
+
+class Map extends Component {
+  constructor () {
+    super();
+    this.map;
```

Comment:
curiously the `this.map` is not working without the constructor this way. I'll check again.